packages/dm-core-plugins/src/list/ListPlugin.tsx

```diff
@@ -284,14 +287,16 @@ export const ListPlugin = (props: IUIPlugin & { config?: TListConfig }) => {
           rowsPerPage={paginationRowsPerPage}
           setRowsPerPage={setPaginationRowsPerPage}
         />
-        <Stack direction="row" spacing={1}>
-          <AppendButton onClick={() => addItem()} />
-          <SaveButton
-            onClick={save}
-            disabled={!unsavedChanges}
-            isLoading={isLoading}
-          />
-        </Stack>
+        {!internalConfig.readOnly && (
```

Comment:
If you need to use list with read-only, please instead use `internalConfig.functionality.add` and wrap the append button in it. For the save button you could check if both functionality.delete, .add, .edit and .sort are false, if so hide the save button

Reply:
Landed on not addressing this in this PR